src/Compiler/Directory.Build.props

```diff
@@ -40,5 +40,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" NoWarn="NU1608" PrivateAssets="All" />
     <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" PrivateAssets="All" />
     <PackageReference Include="Roslyn.Diagnostics.Analyzers" PrivateAssets="All" />
+
+    <Analyzer Include="$(ArtifactsBinDir)Razor.Diagnostics.Analyzers\$(Configuration)\netstandard2.0\Razor.Diagnostics.Analyzers.dll"  />
```

Comment:
Is the DLL built automatically in Visual Studio somehow? Couldn't we instead use `<ProjectReference Include="..." OutputItemType="Analyzer" ReferenceOutputAssembly="false" />`? (Maybe that has its own problems though.)

Reply:
It won't be built automatically in VS, but it's built by running `build.cmd`. I thought that was a required step to get `startvs` to work, which I _think_ most folks are using?

Reply:
I did try adding it as a project reference as you suggested and hit a bunch of issues, especially with the VSIX.

Reply:
I thought the required step is only `restore.cmd`? But anyway, fine by me.

Reply:
Ah, that's probably correct. I'll take a look at updating that script as well.

Reply:
Or, maybe I'll take another stab at just putting the analyzer projects in Razor.sln. :smile:

Reply:
As I said - no need to do anything, I'm fine with building the analyzer before it starts to work... :)

Reply:
Yeah, but I really don't want to break existing developer workflows. :smile:

Reply:
I think I got it working.